common/GL/ContextEGL.cpp

```diff
@@ -285,74 +287,56 @@ namespace GL
 		Console.WriteLn(
 			"Trying version %u.%u (%s)", version.major_version, version.minor_version,
 			version.profile == Context::Profile::ES ? "ES" : (version.profile == Context::Profile::Core ? "Core" : "None"));
-		int surface_attribs[16];
-		int nsurface_attribs = 0;
-		surface_attribs[nsurface_attribs++] = EGL_RENDERABLE_TYPE;
-		surface_attribs[nsurface_attribs++] = (version.profile == Profile::ES) ?
-                                                  ((version.major_version >= 3) ? EGL_OPENGL_ES3_BIT :
-                                                                                  ((version.major_version == 2) ? EGL_OPENGL_ES2_BIT : EGL_OPENGL_ES_BIT)) :
-                                                  EGL_OPENGL_BIT;
-		surface_attribs[nsurface_attribs++] = EGL_SURFACE_TYPE;
-		surface_attribs[nsurface_attribs++] = (m_wi.type != WindowInfo::Type::Surfaceless) ? EGL_WINDOW_BIT : 0;
-		surface_attribs[nsurface_attribs++] = EGL_RED_SIZE;
-		surface_attribs[nsurface_attribs++] = 8;
-		surface_attribs[nsurface_attribs++] = EGL_GREEN_SIZE;
-		surface_attribs[nsurface_attribs++] = 8;
-		surface_attribs[nsurface_attribs++] = EGL_BLUE_SIZE;
-		surface_attribs[nsurface_attribs++] = 8;
-		surface_attribs[nsurface_attribs++] = EGL_NONE;
-		surface_attribs[nsurface_attribs++] = 0;
+
+		const auto surface_attribs = [this, version]() -> std::array<int, 16> {
+			const auto renderable_type_value = (version.profile == Profile::ES) ?
+												   ((version.major_version >= 3) ? EGL_OPENGL_ES3_BIT :
+                                                                                   ((version.major_version == 2) ? EGL_OPENGL_ES2_BIT : EGL_OPENGL_ES_BIT)) :
+                                                   EGL_OPENGL_BIT;
```

Comment:
There seems to be a mix of tabs and spaces, better just stick to tabs

Reply:
The formatting has been done by the project's `.clang-format`! :sweat_smile: (You can see that the previous code was somewhat similar styled)!

I am happy to make a special `clang-format off` for this section, and make the code _stick to tabs_ if needed / if you like.

Reply:
Clang formatting in PCSX2 has a lot to be desired unfortunately. Don't think there is a correct and easy way to go about it.

Reply:
The extra formatting on certain untouched parts of the codebase were on my part (running `clang-format` instead of `git clang-format` sorry!), but all linters are not "optimal" in the end, so happy to just adjust to whatever helpful comments to correct the problems any time.